pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[build-system]
+build-backend = "hatchling.build"
+
+requires = [ "hatchling" ]
```

Comment:
I think this sets this project apart from others. Is this something you want to start using across the board?

Reply:
Frankly, that's just convenience: hatchling created the pyproject.toml file automatically. I can see if I can configure it to use setuptools instead.

Reply:
I haven't used hatchling, so I'm not against it and don't really have any strong opinions on packaging options. So I was mostly curious and I know Jens Erik talked about hatchling a while back.

Reply:
From what I've seen, the discovery is somewhat smarter. You don't need a `MANIFEST.in` file. I guess setuptools grew quite complex over the years. I changed things to setuptools, let's see how it works.